views/js/pciCreator/dev/textReaderInteraction/pciCreator.json

```diff
@@ -0,0 +1,27 @@
+{
+    "typeIdentifier": "textReaderInteraction",
+    "label": "Text reader",
+    "description": "The Paging widget combines a scrolling widget with additional paging controls.",
+    "version": "0.1",
+    "author": "Aleh Hutnikau",
+    "email": "hutnikau@1pt.com",
+    "libraries": [
+        "IMSGlobal/jquery_2_1_1",
+        "./runtime/js/renderer",
+        "./runtime/js/tabs"
+    ],
+    "css": [
+        "./runtime/css/textReaderInteraction.css" 
+    ],
+    "entryPoint": "./runtime/textReaderInteraction.amd.js",
+    "tags": [
+        "reader",
+        "paging"
+    ],
+    "icon": "./creator/media/icon.svg",
+    "short": "Text reader",
+    "response": {
+        "baseType": "integer",
+        "cardinality": "multiple" 
+    }
```

Comment:
the default response declaration should match the getResponse of you pci : if you go for boolean, then use rather : response : {baseType : "boolean", cardinality : "single"}